lib/internal/console/constructor.js

```diff
@@ -61,6 +61,13 @@ const {
 } = require('internal/constants');
 const kCounts = Symbol('counts');
 const { time, timeLog, timeEnd, kNone } = require('internal/util/debuglog');
+const { channel } = require('diagnostics_channel');
+
+const onLog = channel('console.log');
```

Comment:
The build failure comes from exceptions when taking startup snapshots. Tring to fix weak map support in the builtin startup snapshot in https://github.com/nodejs/node/pull/56403.